Файлы, что получились и начало отчета

KlimovaAnna opened the pull request.

KlimovaAnna committed:
Файлы по первому этапу и начало отчета в ворде

KlimovaAnna commented:
Файлы по первому этапу и начало отчета в ворде, спутника номер 6 (как по журналу вариантов) нет... Какой тогда взять? Может 24?

KlimovaAnna committed:
Delete Курсовой часть 1 Климова.docx

KlimovaAnna committed:
Для спутника 24

Korogodin commented:
Если посмотреть отчеты других студентов, то 6 спутник есть

![image](https://user-images.githubusercontent.com/747285/55151902-4c47e080-5160-11e9-9f20-6e37cb77fbb0.png)

KlimovaAnna commented:
В файле нав не нашла... 

Климова Анна Павловна,

Специалист Отдела международного сотрудничества

ФГБОУ ВО Национальный исследовательский университет "МЭИ"


четверг, 28 марта 2019 г., 13:49 +0300 от notifications@github.com  <notifications@github.com>:
>Если посмотреть отчеты других студентов, то 6 спутник есть
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  mute the thread .

Korogodin commented:
В nav-файле действительно может отсутствовать запись, но она там дублирует таблицу. Надо в отчете привести выдержку из nav-файла и написать комментарий.

KlimovaAnna commented:
Выдержку о том, что его нет?

Климова Анна Павловна,

Специалист Отдела международного сотрудничества

ФГБОУ ВО Национальный исследовательский университет "МЭИ"


четверг, 28 марта 2019 г., 14:06 +0300 от notifications@github.com  <notifications@github.com>:
>В nav-файле действительно может отсутствовать запись, но она там дублирует таблицу. Надо в отчете привести выдержку из nav-файла и написать комментарий.
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  mute the thread .

Korogodin commented:
Выдержку из файла. Пусть там нет этого спутника, но другие есть. Можно будет хоть обсудить формат файла при защите

KlimovaAnna commented:
Вы имеете в виду вот так скриншотом?




>Четверг, 28 марта 2019, 15:06 +03:00 от Korogodin Ilya <notifications@github.com>:
>
>Выдержку из файла. Пусть там нет этого спутника, но другие есть. Можно будет хоть обсудить формат файла при защите
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  mute the thread .

С уважением,
-- 
Климова Анна Павловна,
Специалист Отдела международного сотрудничества
ФГБОУ ВО Национальный исследовательский университет "МЭИ"
__________________________________________________________
Klimova Anna
Specialist of Department of International Relations
National Research University "Moscow Power Engineering Institute"

Korogodin commented:
Хотя бы скриншотом

KlimovaAnna committed:
Delete Курсовой часть 1 Климова.docx

KlimovaAnna committed:
Добавила скриншоты двух .nav файлов

Korogodin commented:
Обращаю внимание, что задание курсового проекта прошлого года и этого года отличаются. То, что представлено в первой половине отчета, соответствует заданию прошлого года - RTKPOST, расчет координат и т.п. Этого в задании сейчас нет. В этом году нужно было с помощью программы RTKNAVI получить эфемериды собственного спутника, которые будут использоваться на следующих этапах. Судя по отчету, эта задача не выполнена.

KlimovaAnna committed:
Delete Курсовой часть 1 Климова.docx

KlimovaAnna committed:
Исправила вступление

Все остальное делала по заданию изначально. К примеру из одного исходного файла для вечера получила три файла, один из которых как раз нави. Нашла свой спутник, потом провела первичное моделирование в Тримбл.

Korogodin commented:
У меня ощущение, что у нас недопонимание. 
Задание с srns.ru:

Оформить отчёт по результатам этапа:

1. Описание процесса использования RTKLIB
2. Эфемериды собственного спутника по данным RTKNAVI из состава RTKLIB (номер спутника GPS см. в журнале выше)
3. Эфемериды собственного спутника в nav-файле RINEX
4. График угла места собственного спутника от времени по данным Trimble GNSS Planning Online на заданный интервал времени (см. задание второго этапа)
5. SkyView по данным Trimble GNSS Planning Online на заданный интервал времени (см. задание второго этапа)

Сейчас не хватает пункта 2 - таблички из RTKNAVI с эфемеридами для 6 спутника. Её скриншот я прикладывал выше.

KlimovaAnna committed:
Delete Курсовой часть 1 Климова.docx

KlimovaAnna committed:
Готово

Все, теперь все красиво исправила, расписала по пунктам и добавила недостающие скриншоты.

Korogodin commented:
Вообще надо было это сделать для 6 спутника, но да ладно.
